dev/io.openliberty.jakarta.xmlBinding.4.0/src/io/openliberty/jakarta/xmlbinding/JaxbToolsUtil.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
copyright should be 2022.  Why are these classes here instead of a io.openliberty.xmlbinding.tools project like the comment below?